### PR TITLE
[CNFT1-3364-panel-button-fixes] changing reset button styles and adding panel min-width

### PR DESCRIPTION
**Pull request by hclarkEnq**
Issue

Text on the button is misaligned
Screenshot 2024-10-14 at 6 44 12 AM

To fix this I added the correct styles for the panel expected matching the Designs in Figma
Screenshot 2024-10-14 at 6 46 13 AM

* [CNFT1-3364](https://cdc-nbs.atlassian.net/browse/CNFT1-3364)
Figma https://www.figma.com/design/TVilgvbjEwoYTqtQekVZwJ/NBS-7.x--%3E-Patient-Search-2.0?node-id=17-10016&node-type=canvas&m=dev
Screenshot 2024-10-14 at 6 41 39 AM

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-3364]: https://cdc-nbs.atlassian.net/browse/CNFT1-3364?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss [hunk 1]
```diff
@@ -29,6 +30,12 @@
         gap: 1rem;
 
         @extend %thin-top;
+
+        .reset {
+            padding: 0;
+            font-size: 0.875rem;
+            min-width: auto;
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss:37]: suggestion:  Will need to double check with the design team as to why this button is different and if that was intentional.

File: apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 .panel {
     display: flex;
     flex-direction: column;
+    min-width: 15.5rem;
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss:7]: The min-width for this panel is currently coming from [here](https://github.com/CDCgov/NEDSS-Modernization/blob/9fb7f87a88dd212763506763b085e95273057ad3/apps/modernization-ui/src/apps/search/layout/result/header/options/table/search-results-table-options.module.scss#L4).  With the reasoning being that the Column preference panel could be any size but for these table options it's in an overlay panel of a certain size.

Comment (hclarkEnq) [reply apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss:7]: Okay makes sense

**Review by alaapbharadwaj — APPROVED** [review]

File: apps/modernization-ui/src/components/button/Button.module.scss [hunk 3]
```diff
@@ -17,3 +17,11 @@
     line-height: 162%;
     font-style: normal;
 }
+
+.unstyled {
+    text-decoration: none !important;
+
+    &:hover {
+        text-decoration: underline !important;
+    }
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/components/button/Button.module.scss:26]: issue: Unstyled button is not present in our design system so we default to [USWDS](https://designsystem.digital.gov/components/button/) which has it styled like a link.

File: apps/modernization-ui/src/design-system/table/preferences/ColumnPreferencesPanel.tsx [hunk 4]
```diff
@@ -103,7 +103,7 @@ const ColumnPreferencesPanel = ({ close }: Props) => {
                 </Droppable>
             </DragDropContext>
             <footer>
-                <Button unstyled onClick={handleReset}>
+                <Button className={styles.reset} unstyled onClick={handleReset}>
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/table/preferences/ColumnPreferencesPanel.tsx:106]: issue:  this is not an unstyled button

**Review by adamloup-enquizit — COMMENTED** [review]
The reset button is not an unstyled button, it has differing styles.

**Review by adamloup-enquizit — APPROVED** [review]
👍

**Review by joeaxcdw — APPROVED** [review]

**Pull request merged**